recipes/sagelib/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
+export CFLAGS="-w $CFLAGS"
+export CXXFLAGS="-w $CXXFLAGS"
+
+export SAGE_FAT_BINARY=yes
+export SAGE_LOCAL=$PREFIX
+ln -s $PREFIX local
+export SAGE_NUM_THREADS=2
+
+# make build builds sagelib and copies some scripts
+# the scripts can be moved to another package and call it sageruntime
+make build
```

Comment:
Is that possible? I thought there were files generated by the `Makefile`. If they are not needed for sagelib, we can just use `setup.py`